tool/mesos/Makefile

```diff
@@ -0,0 +1,25 @@
+CXX_FLAGS=-I ../include -std=c++11 -I /usr/local/include/hdfs -I /root/incubator-singa/include
```

Comment:
since mesos is placed inside incubator-singa,
we can replace `/root/incubator-singa` with `../../include`.